modules/end-user-guide/partials/proc_using-a-git-credentials-store.adoc

```diff
@@ -75,11 +74,13 @@ data:
 You can create and apply multiple Git credentials Secrets in your user {orch-namespace}. All of them will be copied into one Secret that will be mounted to the `{devworkspace}` containers. For example, if you set the mount path to `/etc/secret`, then the one Secret with all of your Git credentials will be mounted at `/etc/secret/credentials`. You must set all Git credentials Secrets in your user {orch-namespace} to the same mount path. You can set the mount path to an arbitrary path because the mount path will be automatically set in the Git configuration file configured at `/etc/gitconfig`.
 ====
 
-. Apply the Secret:
+. Apply the Secret.
 +
-[subs="+quotes,+attributes,+macros"]
+[source,terminal,subs="+quotes,+attributes"]
```

Comment:
Not a source code block.
```suggestion
[subs="+quotes,+attributes"]
```

Reply:
Robert, it was my suggested change. This is what `source,terminal` does--see the upper right corner of the block:
![image](https://user-images.githubusercontent.com/78564304/180978180-3e5918f3-671d-4226-a209-ce3010b46c0a.png)

Reply:
That's two UX benefits:
1. Convenience of copypaste.
2. It's clear that's in the terminal.

Reply:
Here's the link to that page:
https://www.eclipse.org/che/docs/stable/administration-guide/upgrading-che-7-41-on-openshift/

Reply:
I don't think that justifies it. First, it's not semantic. But OK, that's splitting hairs. But I don't think the copy button is so useful (given that it would copy the content together with the prompt (`$`), meaning some editing would still be required.

That said, I'm not dead set on this, and I don't want to spend too much time arguing over it.